### ISSUE TITLE
MAGeT.py: input files in native space?

### ISSUE THREAD

**Issue by gdevenyi** [Mouse-Imaging-Centre/pydpiper#195]
I have a complete MAGeT.py run that ran with no issues on scinet (yay) however the final voted labels seem to be all wrong.

The labels when displayed on the input brain don't seem to be anywhere near the subject brain.

This is the command I used to start the run
```
MAGeT.py --num-executors=31 \
    --verbose --pipeline-name=D2KO-MAGET \
    --atlas-library ./atlas \
    --lsq12-protocol ./default_linear_MAGeT_prot.csv \
    --nlin-protocol ./default_nlin_MAGeT_minctracc_prot.csv \
    --max-templates 21 \
    --config-file /project/j/jlerch/matthijs/pydpiper_modules/pydpiper-master-v1.13.1/python/pydpiper-1.13.1-py2.7.egg/config/SciNet.cfg \
    $(for line in $(cat inputs/templates.txt); do ls inputs/*$line* | grep D2K0; done) inputs/*.mnc
```

I'm using the c57_average.mnc atlas.

The help from MAGeT.py says:
```
  --input-space INPUT_SPACE
                        Option to specify space of input-files. Can be native
                        (default), lsq6, lsq12.
```

However elsewhere on the (older) wiki: 
"Alignment
This code assumes that all brains (both atlases and inputs) are already more or less in the same space. In nomenclature many of us at MICe are familiar with, this means that they are already in LSQ6 space. Eventually, the MAGeT code will do this alignment for you, if needed. For now, brains that have undergone six-parameter alignment must be used."

I assumed the help output was up-to-date, but I guess I need my subjects in the atlas space before I start?

**Comment by mcvaneede** [comment]
Correct. The --input-space flag should actually not even be available for MAGeT.py. The way we have always run MAGeT is either on the input files after they have been aligned to target space using a rigid transformation, or on the final average of a pipeline (also rigidly aligned to atlas space). We have our initial models all in atlas space, and thus after running a registration pipeline we have the files where we want them.

MAGeT currently has no way to deal with files that have not been rigidly aligned to the atlas space.

**Comment by gdevenyi** [comment]
Respinning as BUG for documentation.

**Comment by gdevenyi** [comment]
Still listed as an option for MAGeT in 1.14

**Comment by bcdarwin** [comment]
The 2.x code only uses --input-space for the registration chain (you can disable lsq6 with --no-lsq6, for instance; we could have something similar for lsq12).  Do we even want this flag?
